### ISSUE TITLE
Detected I/O inside event loops

### ISSUE THREAD

**Issue by rgruyters** [xirixiz/Home-Assistant-Sensor-Afvalwijzer#57]
With the coming release of 0.109.0 Home Assistant has included logic to detect blocking calls.

```
Detected I/O inside the event loop. This is causing stability issues. Please report issue to the custom component author for afvalwijzer doing I/O at custom_components/afvalwijzer/sensor.py, line 61: afvaldienst = Afvaldienst(provider, zipcode, housenumber, suffix, count_today)
Detected I/O inside the event loop. This is causing stability issues. Please report issue to the custom component author for afvalwijzer doing I/O at custom_components/afvalwijzer/sensor.py, line 159: afvaldienst = Afvaldienst(provider, zipcode, housenumber, suffix, count_today)
```

**Comment by xirixiz** [comment]
Thanks for the heads up! I`ll look into it!

**Comment by xirixiz** [comment]
Not sure how to solve this yet. I know how to do some things with Python, but I`m not a die hard programmer. I hope I`ll figure out soon based on other ppls code or someone creates a PR.

**Comment by tomlevels** [comment]
I see a similar issue fixed here, maybe it can help you to fix it:
https://github.com/home-assistant/core/pull/34442

**Comment by YRRK** [comment]
It's new since HomeAssistant 109
https://www.home-assistant.io/blog/2020/04/29/release-109/

I have the message too :-)

2020-04-29 20:29:43 WARNING (MainThread) [homeassistant.util.async_] Detected I/O inside the event loop. This is causing stability issues. Please report issue to the custom component author for afvalwijzer doing I/O at custom_components/afvalwijzer/sensor.py, line 61: afvaldienst = Afvaldienst(provider, zipcode, housenumber, suffix, count_today)

2020-04-29 20:30:15 WARNING (MainThread) [homeassistant.util.async_] Detected I/O inside the event loop. This is causing stability issues. Please report issue to the custom component author for afvalwijzer doing I/O at custom_components/afvalwijzer/sensor.py, line 159: afvaldienst = Afvaldienst(provider, zipcode, housenumber, suffix, count_today)

**Comment by rgruyters** [comment]
So far I can see this has been solved in the latest version of Afvalwijzer, 4.2.0.
